openspec/changes/add-company-selection-persistence/tasks.md

```
@@ -0,0 +1,12 @@
+## 1. Implementación
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)